staging_service/app.py

```diff
@@ -50,6 +53,20 @@
 }
 
 
+@routes.get("/importer_filetypes/")
+async def importer_filetypes(request: web.Request) -> web.json_response:
+    """
+    Returns the file types for the configured datatypes. The returned JSON contains two keys:
+    * datatype_to_filetype, which maps import datatypes (like gff_genome) to their accepted
+      filetypes (like [FASTA, GFF])
+    * filetype_to_extensions, which maps file types (e.g. FASTA) to their extensions (e.g. 
+      *.fa, *.fasta, *.fa.gz, etc.)
+
+    This information is currently static over the life of the server.
+    """
+    return web.json_response(data=_DATATYPE_MAPPINGS)
```

Comment:
Will this return anything if  `importer_mappings` isn't called to populate that variable?

Reply:
Not quite sure I understand - `_DATATYPE_MAPPINGS` is populated on startup via the call stack
```
__main__.py -> app_factory -> inject_config_dependencies
```
It doesn't depend on `import_mappings` to be populated. Maybe I don't understand what you're saying?

Reply:
Ah, the way the github preview made it seem like the code was in the importer_mappings function, which got me confused. Nevermind